taps_to_cases/runner/src/main/java/com/microsoft/hydralab/t2c/runner/controller/BaseDriverController.java

```diff
@@ -22,9 +22,11 @@
 public abstract class BaseDriverController {
     protected WebDriver webDriver;
     protected Logger logger;
+    protected String udid;
 
-    public BaseDriverController(WebDriver webDriver, Logger logger) {
+    public BaseDriverController(WebDriver webDriver, String udid, Logger logger) {
```

Comment:
What's the usage of udid here? "uuid"?

Reply:
T2C can run well with driverId rather than UDID. But for the perf implements, we need udid to identify which device we want to monitor if there are multiple devices required in the json file